### PR TITLE
Support for ShardingSphere 5.0.0

### PR DESCRIPTION
**Pull request by dachuan9e**
Support for ShardingSphere 5.0.0

### CODE REVIEW

**Comment by wu-sheng** [comment]
The changes.md in the root should be updated

File: test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list [hunk 1]
```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-5.0.0-beta
+5.0.0
```
Comment (wu-sheng) [inline test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list:17]: Is `beta` still supported? Or it is out of the supported list. Either way, please make it clear in the change log.

**Comment by wu-sheng** [comment]
It seems you breaks something, please recheck your codes.

File: test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list (hunk 1 above)
Comment (dachuan9e) [reply test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list:17]: Beta can be supported too.

Comment (dachuan9e) [reply test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list:17]: However, we don't recommend the beta version.

Comment (wu-sheng) [reply test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list:17]: OK, just make sure it is clear in the change log. The version recommendation is your community's call for sure.

**Comment by wu-sheng** [comment]
I fixed the existing cases. Please fix conflicts.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Thanks for your contribution.

**Pull request merged**